packages/mui-material/src/styles/ThemeProvider.tsx

```diff
@@ -36,6 +36,12 @@ export interface ThemeProviderProps<Theme = DefaultTheme> extends ThemeProviderC
    * @default document
    */
   documentNode?: Document | null;
+  /**
+   * The default mode when the storage is empty,
```

Comment:
What "storage" means? Could we specify a bit more?

Reply:
May be `local storage` is more direct?

Reply:
👍